WeakAuras/WeakAuras.lua

```diff
@@ -4610,7 +4609,7 @@ function WeakAuras.PerformActions(data, type, region)
     return;
   end
 
-  if(actions.do_message and actions.message_type and actions.message and not squelch_actions) then
+  if(actions.do_message and actions.message_type and actions.message) then
```

Comment:
This squelch_actions was originally to prevent a huge burst of sounds if a user logged in a the wrong time, if i remember correctly. Is that now possible?

Reply:
thanks for giving context i didn't know what it was

Reply:
Yes, I've kept it around for sound but removed it for chat and actions.

Reply:
Great, thanks.